### PR TITLE
initial setup for react lite

### PR DESCRIPTION
**Pull request by monty2610**
This setup contain initial setup for react lite. Structure is copied from create-react-app and configurations are added from universal-react. 
Redux/saga integration is pending

### CODE REVIEW

File: .1.gitignore [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.DS_Store
```
Comment (vinodloha) [inline .1.gitignore]: Looks like a duplicate file, Please keep the most correct one.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,44 @@
-# react-lite
-Client side React and Redux based development framework
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
```
Comment (vinodloha) [inline README.md]: We should be suing yarn

File: README.md [hunk 3]
```diff
@@ -1,2 +1,44 @@
...
+
+Builds the app for production to the `build` folder.<br>
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.<br>
+Your app is ready to be deployed!
+
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+### `npm run eject`
```
Comment (vinodloha) [inline README.md]: This documentation is of 
`npm run eject`
 really needed, We should document Scripts such as 
`yarn run storybook`
`yarn run generate`

File: README.old.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+# react-lite
```
Comment (vinodloha) [inline README.old.md]: Remove this one and move its content to the above. We should customise the documentation in Readme file to meet our structure and don't keep the create react app content.

File: docs/ast/source/.external-ecmascript.js.json [hunk 5]
```diff
@@ -0,0 +1,2802 @@
+{
```
Comment (vinodloha) [inline docs/ast/source/.external-ecmascript.js.json]: @monty2610 content below docs/ should be under gitignore

Anything that can be generated should be under gitignore

File: flow-typed/npm/@babel/core_vx.x.x.js [hunk 6]
```diff
@@ -0,0 +1,305 @@
+// flow-typed signature: 2fd472879d1107760fb18be85d0329a2
```
Comment (vinodloha) [inline flow-typed/npm/@babel/core_vx.x.x.js]: flow-typed/npm/ should be under gitignore

File: reports/flow/assets/codemirror-javascript-mode.js [hunk 7]
```diff
@@ -0,0 +1,752 @@
+// CodeMirror, copyright (c) by Marijn Haverbeke and others
+// Distributed under an MIT license: http://codemirror.net/LICENSE
+
+// TODO actually recognize syntax of TypeScript constructs
+
+(function(mod) {
```
Comment (vinodloha) [inline reports/flow/assets/codemirror-javascript-mode.js]: reports/ should be gitignore

File: .1.gitignore (hunk 1 above)
Comment (monty2610) [reply .1.gitignore]: Done

File: README.md (hunk 2 above)
Comment (monty2610) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (monty2610) [reply README.md]: Done

File: README.old.md (hunk 4 above)
Comment (monty2610) [reply README.old.md]: Done

File: docs/ast/source/.external-ecmascript.js.json (hunk 5 above)
Comment (monty2610) [reply docs/ast/source/.external-ecmascript.js.json]: Done

File: flow-typed/npm/@babel/core_vx.x.x.js (hunk 6 above)
Comment (monty2610) [reply flow-typed/npm/@babel/core_vx.x.x.js]: Done

File: reports/flow/assets/codemirror-javascript-mode.js (hunk 7 above)
Comment (monty2610) [reply reports/flow/assets/codemirror-javascript-mode.js]: Done

**Comment by monty2610** [comment]
Hi @vinodloha , all review comments are already incorporated by @chandrakailash1993. Please check and let us know if any other change needed in this branch

**Pull request merged**